examples/call-client-functions-with-arguments/call-client-functions-with-arguments.js

```diff
@@ -0,0 +1,14 @@
+import { ClientFunction } from 'testcafe';
+
+fixture `Parameterized Client Functions`
+    .page('https://devexpress.github.io/testcafe/example/');
+
+const getLocationPart = ClientFunction(locationPart => {
+    return window.location[locationPart];
+});
+
+test('Parameterized Client Functions', async t => {
+    await t
+        .expect(getLocationPart('host')).eql('devexpress.github.io')
```

Comment:
I see it again and think that this example should be moved to the documentation.

Add `Client function with arguments` paragraph [here] (https://devexpress.github.io/testcafe/documentation/guides/basic-guides/obtain-client-side-info.html#execute-client-functions)